### DISCUSSION TITLE
Can't run any of the DBT Power User features becase it can't read the Big-Query Profile

### DISCUSSION THREAD

**Discussion by EduardoLooma** [AltimateAI/vscode-dbt-power-user#142] [Q&A] [answered]
I'm trying to run some of the DBT Power User features, but it always gives me the same error
`Credentials in profile "bigquery-analytics-dev," target "dev" invalid: Runtime Error Could not find adapter type bigquery!`

My profile is set up correctly since all checks pass when I run the `dbt debug`. Is there something that needs to be changed in the profile, or what is why DBT Power User can't use my credentials?

Thank you

**Comment by mdesmet** [comment, marked as answer]
This is a standard dbt error. My guess is that you don't have `dbt-bigquery` installed in the selected Python environment.

**Comment by EduardoLooma** [comment]
Thank you, it was a problem with the interpreter, I followed the instruction in this [post](https://github.com/innoverio/vscode-dbt-power-user/issues/87#issuecomment-954885093)  and it worked
